### PR TITLE
Get rid of SidePanel

### PR DESCRIPTION
**Pull request by ltclmbdev**
- got rid of SidePanel (once it should the part of outside app)
- moved CueNote to the separate component

Also tried to fix the situation when popups of first cues overlap the player edges, but let it be in next PR.

### CODE REVIEW

File: docs/lib/examples/import-basic.js [hunk 1]
```diff
@@ -1,27 +1,23 @@
 import React from 'react'
 import {Player} from 'cueplayer-react'
 import {PlayerProvider} from 'cueplayer-react/context/player-context'
-import SidePanel from '../Components/side-panel/SidePanel'
```
Comment (joelhooks) [inline docs/lib/examples/import-basic.js]: this sidepanel was just an example implementation and not part of the library

Comment (joelhooks) [reply docs/lib/examples/import-basic.js]: it's demonstrating the capability with the provider

**Review by joelhooks — APPROVED** [review]

**Comment by ltclmbdev** [comment]
Applied animation to the bottom when autoHide occurs: https://share.getcloudapp.com/6qurBLzL

File: docs/lib/examples/import-basic.js (hunk 1 above)
Comment (ltclmbdev) [reply docs/lib/examples/import-basic.js]: put SidePanel stuff back

**Pull request merged**